src/quo2/components/settings/reorder_item/view.cljs

```diff
@@ -0,0 +1,44 @@
+(ns quo2.components.settings.reorder-item.view
+  (:require [quo.react-native :as rn]
```

Comment:
quo shouldn't be used, react-native ns should be used instead

Reply:
Done